recipe/meta.yaml

```diff
@@ -25,8 +25,9 @@ requirements:
     - python >=3.6
     - setuptools
     - flask >=1.0.4
-    - flask-compress
     - plotly >=5.0.0
+    - importlib-metadata ==4.8.3
+    - contextvars ==2.4
```

Comment:
Hard pins are not going to work well in conda environments. I don't' think we should do this.

Reply:
This https://github.com/plotly/dash/blob/v2.9.3/requires-install.txt shows:

```
Flask>=1.0.4
plotly>=5.0.0
dash_html_components==2.0.0
dash_core_components==2.0.0
dash_table==5.0.0
importlib-metadata==4.8.3;python_version<"3.7"
contextvars==2.4;python_version<"3.7"
```

Which seems to indicate that these two packages are only needed for Python < 3.7. Conda forge doesn't build for ython < 3.6, so we shouldn't need them.

Reply:
This is the CI error:

```
The reported errors are:
- Encountered problems while solving:
-   - package contextvars-2.4-py_0 requires python >=3,<3.7, but none of the providers can be installed
```